src/generate-sdk/sdkGenerator/TsSdkGenerator.ts

```diff
@@ -0,0 +1,251 @@
+
+import { AstNodeType, ClassDefinition, MethodDefinition, NativeType, ParameterDefinition, SdkGeneratorInput, SdkGeneratorInterface, SdkGeneratorOutput, TypeAlias, TypeDefinition } from "../../models/genezioModels";
+import { TriggerType } from "../../models/yamlProjectConfiguration";
+import { browserSdkTs } from "../templates/browserSdkTs";
+import { nodeSdkTs } from "../templates/nodeSdkTs";
+
+
+class SdkGenerator implements SdkGeneratorInterface {
+    externalTypes: TypeDefinition[] = [];
+    classImplementation = "";
+
+  async generateSdk(
+      sdkGeneratorInput: SdkGeneratorInput
+    ): Promise<SdkGeneratorOutput> {
+  const options = sdkGeneratorInput.sdk.options;
+  const nodeRuntime = options.runtime;
+  const generateSdkOutput: SdkGeneratorOutput = {
+    files: []
+  };
+  for (const classInfo of sdkGeneratorInput.classesInfo) {
+    this.externalTypes = [];
+    this.classImplementation = "";  
+    const _url = "%%%link_to_be_replace%%%";
+    const classConfiguration = classInfo.classConfiguration
+
+    let classDefinition: ClassDefinition | undefined = undefined;
+
+    if (classInfo.program.body === undefined) {
+      continue;
+    }
+    for (const elem of classInfo.program.body) {
+      if (elem.type === AstNodeType.ClassDefinition) {
+        classDefinition = elem;
+      }
+    }
+
+    if (classDefinition === undefined) {
+      continue;
+    }
+    this.classImplementation = `/**
```

Comment:
To be consistent with the other generators, this should also be `const template = `